SPECS/tpm2-pytss/tpm2-pytss.spec

```diff
@@ -0,0 +1,66 @@
+Name:       tpm2-pytss
+Version:    1.2.0
+Release:    3%{?dist}
+Summary:    Python bindings for tpm2-tss
+License:    BSD
+URL:        https://github.com/tpm2-software/tpm2-pytss
+Group:      System Environment/Security
+Vendor:     Microsoft Corporation
+Distribution: Mariner
+
+Source0: https://github.com/tpm2-software/tpm2-pytss/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+
+BuildRequires: python3-devel
+BuildRequires: python3-setuptools
+BuildRequires: python3-pkgconfig
+BuildRequires: python3-pycparser
+BuildRequires: python3-pip
+BuildRequires: git
+BuildRequires: tpm2-tss-devel
+
+BuildRequires: python3-packaging
```

Comment:
so, this section of buildrequires: doesn't seem to come from the photon spec: https://github.com/vmware/photon/blob/master/SPECS/tpm2-pytss/tpm2-pytss.spec#L14

did this come from somewhere else?

Reply:
i added them because i was seeing build failures locally and saw those BuildRequires in other py packages.

Reply:
checking what happens without this change now

Reply:
see error in build with just the photon bits:
```
distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmp52wb7ip3', '--quiet', 'packaging']' returned non-zero exit status 1.
```

Reply:
after adding python3-packaging, this error:
```
distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpmy4jm0zw', '--quiet', 'asn1crypto']' returned non-zero exit status 1.
```

Reply:
after adding python3-asn1crypto

```
distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmp8t6rdhd5', '--quiet', 'cryptography>=3.0']' returned non-zero exit status 1.
```

Reply:
after adding python3-cryptography
```
distutils.errors.DistutilsError: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmprrvxin45', '--quiet', 'setuptools_scm[toml]>=3.4.3']' returned non-zero exit status 1.
```

Reply:
seems like each is required, i can add a comment

Reply:
sounds great, thanks!